src/javascripts/views/loginView.js

```diff
@@ -0,0 +1,58 @@
+import { MESSAGE, ERROR_MESSAGE } from "../constants/message";
+import { isRequired, isValidForm } from "../helpers/validationForm";
+
+class LoginView{
+    
+    constructor(){
+        this.loginForm = document.querySelector(".login-form");
+        this.buttonElement = document.querySelector(".login-form-button");
+    }
+
+    //----- EVENT LISTENER -----//
+    addEventLogin = (isValidAccount) => {
+        this.buttonElement?.addEventListener("click", async(e)=>{
+            e.preventDefault();
+
+            const accountInput = {
+                email: this.loginForm.email.value,
+                password: this.loginForm.password.value,
+            }
+
+            const emailError = document.querySelector(".error-email");
+            const passwordError = document.querySelector(".error-password");
+            const errorCredentials = document.querySelector(".error-credentials");
+
+            const errorMessages = [];
+
+            if (!isRequired(accountInput)){
+              errorMessages.push(MESSAGE.EMAIL_REQUIRED, MESSAGE.PASSWORD_REQUIRED);
+            }
```

Comment:
Should be

if () {
 // code
} else if () {
 // code 2
} else {
  // code 3
}